src/java/test/org/apache/zookeeper/test/GetEphemeralsTest.java

```diff
@@ -0,0 +1,234 @@
+/**
```

Comment:
this file should be moved over to zookeeper-server :)

Reply:
Good catch:). We will have to rebase the package restructure change in our internal version.